### PR TITLE
Fix: Double echo

### PR DESCRIPTION
**Pull request by joeflack4**
## Changes
    Resolved double echo
    - UX improvement / fix of logging from mondo-ingest.Makefile, where echo's would be printed twice because they did not start with @.

---

CC @souzadevinicius

### CODE REVIEW

File: src/ontology/mondo-ingest.Makefile [hunk 1]
```diff
@@ -400,7 +400,7 @@ ALL_COMPONENT_SIGNTAURE_REPORTS=$(foreach n,$(ALL_COMPONENT_IDS), reports/mirror
 
 .PHONY: signature_reports
 signature_reports: $(ALL_MIRROR_SIGNTAURE_REPORTS) $(ALL_COMPONENT_SIGNTAURE_REPORTS)
-	echo "Finished running signature reports.."
```
Comment (joeflack4) [inline src/ontology/mondo-ingest.Makefile:403]: Also removed a second `.` here

File: src/ontology/mondo-ingest.Makefile [hunk 2]
```diff
@@ -336,15 +336,15 @@ build-mondo-ingest:
 		mapped-deprecated-terms mapping-progress-report \
 		recreate-unmapped-components sync documentation \
 		prepare_release
-	echo "Mondo Ingest has been fully completed"
+	@echo "Mondo Ingest has been fully completed"
```
Comment (joeflack4) [inline src/ontology/mondo-ingest.Makefile:339]: Example of one of the changes

Comment (joeflack4) [inline src/ontology/mondo-ingest.Makefile:339]: ## ODK PR
Note also that the same issue also exists in the `Makefile`, e.g. `echo Finished running all tests successfully.`, which comes from ODK. So I'd like to do the same quick PR in ODK as well.

**Review by matentzn — APPROVED** [review]
Good move, thanks!

**Pull request merged**